libraries/AP_HAL_ChibiOS/hwdef/ReaperF745/hwdef.dat

```diff
@@ -121,6 +121,7 @@ define HAL_LOGGING_DATAFLASH_ENABLED 1
 # spi devices bmi270(v2) icm42688 (v3)
 SPIDEV bmi270    SPI3 DEVID1 GYRO_CS  MODE3  1*MHZ  4*MHZ
 SPIDEV icm42688  SPI3 DEVID1 GYRO_CS  MODE3  2*MHZ  8*MHZ
+SPIDEV mpu6000   SPI3 DEVID1 GYRO_CS  MODE3  1*MHZ  4*MHZ
```

Comment:
4Mhz is pretty slow. I would stick with 8Mhz if that works ok

Reply:
sorry to go dark. I copied that line from another board. I'll try the IMU at 8k. I haven't gotten my build air born yet, but I will report back after I am able to flight test.

Reply:
ping @ericjohnson97 how'd the flight test go?

Reply:
I've run into a number of issues that I haven't worked through yet. First, I took my new drone out to test at the beginning of august in Texas and after a bit of time in the sun and on the hot concrete the 3d printed frame started melting. 
😂

I also was unable to get a GPS lock even though the board reported detecting a ublox gps.  I reprinted my drone in ASA and went to a GPS module i've had success with in the past.  I took it on the roof of my apartment to see if I could get a GPS lock this weekend and unfortunately it still was not getting a GPS lock even though once again it detected a ublox GPS. 

This time I did notice that after powering on the drone the estimated velocity was growing bigger and bigger over time. I re-did all of the calibration, but no luck. I was going to reflash it and start the setup from scratch again, but I've been a little busy. 

Let me know if you have any advice for those issues. I'll report beck once I hopefully get my aircraft in the air. 

here are some pics of my setup for context
![20240802_231510](https://github.com/user-attachments/assets/0a413c96-fa8a-445b-9550-ecd6740a1f28)
![20240802_231600](https://github.com/user-attachments/assets/8deabc91-14a2-4a29-ac39-f37aff4b69ad)
![20240802_231437](https://github.com/user-attachments/assets/4f32dd4f-c535-48ca-a03a-ba8cd615bf26)
![20240731_235845](https://github.com/user-attachments/assets/d0948ca7-73a0-4cf9-b1c6-aa8a1183dda6)

I'm using OpenHD for video and telemetry.